paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -846,14 +847,14 @@ def autoscaling_is_paused():
         return False
 
 
-def is_autoscaling_resumed(formatted_application: Union[V1Deployment, V1StatefulSet]):
-    if formatted_application.metadata.annotations:
-        if "is_paused" in formatted_application.metadata.annotations:
-            return formatted_application.metadata.annotations["is_paused"] == "False"
-    return True
+def is_deployment_marked_paused(
+    kube_client: KubeClient, service_config: KubernetesDeploymentConfig
+):
+    annotations = get_annotations_for_deployment(kube_client, service_config)
+    return annotations.get("is_paused", False) == "True"
```

Comment:
This line is confusing at first glance, since you're comparing a bare literal `False` to a stringified `"True"`.  I suggest making the default `False` a string, and leaving a comment that the annotations from k8s are strings.